server/routes/schools.js

```diff
@@ -0,0 +1,67 @@
+var express = require('express');
+var router = express.Router();
+var bodyParser = require("body-parser");
+
+const { PrismaClient } = require('@prisma/client')
+
+router.use(bodyParser.json());
+router.use(bodyParser.urlencoded({ extended: false }));
+
+async function getSchools(){
+  const allMySchools = await prisma2.school.findMany();
+  return allMySchools
+}
+
+async function postSchools(body){
+  let name = body.name;
+  let address = body.address;
+  let numberOfStudents = body.numberOfStudents;
+  let specialityPrograms = body.specialityPrograms;
+
+  if(!specialityPrograms || !name || !address || !numberOfStudents ){
+    return 422;
```

Comment:
Returning 400 for [HTTP 400 Bad Request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400) is also valid here. Either should be fine for us though